src/index.js

```diff
@@ -4,28 +4,90 @@ import './style.css';
 // Name and score selectors from the form field to be entered.
 const form = document.querySelector('.form-input');
 const [name, score] = form.elements;
+
+// Refresh button from tag
+const btnRefresh = document.querySelector('.btn-refresh');
+const responsePost = document.querySelector('.response-post');
+
+// New Player object created
 const objPlayers = new Players();
 
+// Assigning the API endpoint to a variable for use.
+const urlNewGame = `
+  https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/`;
+const nameGame = { name: "Bamidele's cool name" };
+
 // Local Storage condition to store players and input
 if (localStorage.savedPlayers) {
   objPlayers.players = JSON.parse(localStorage.getItem('savedPlayers'));
 }
 
+const keyNewGameAPIs = async () => {
+  const response = await fetch(urlNewGame, {
+    method: 'POST', // *GET, POST, PUT, DELETE, etc.
+    mode: 'cors', // no-cors, *cors, same-origin
+    body: JSON.stringify(nameGame),
+    cache: 'no-cache', // *default, no-cache, reload, force-cache, only-if-cached
+    credentials: 'same-origin', // include, *same-origin, omit
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  const data = await response.json();
+  return data.result.slice(14, 34);
+};
+
+const keyGame = keyNewGameAPIs();
+
+// Getting THE API from the endpoint
+const getAPIs = async () => {
+  const response = await fetch(`
+    https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${keyGame}/scores`);
+  const data = await response.json();
+  objPlayers.players = data.result;
+  objPlayers.playerDisplay();
+  return data.result;
+};
+
```

Comment:
- [ ] While I don't receive the same error as the previous reviewer when adding a score I am now unable to get the scores added. I added different scores but they are not fetched and displayed when I click the refresh button or reload the page. Kindly fix this.

- [ ] **[OPTIONAL]** I would also recommend you fix your app. It displays a lot of unnecessary `undefined` name.

  ![rdun](https://user-images.githubusercontent.com/36542986/195554659-3c13b108-0fcf-4d6d-ad9f-2b620c50d914.jpg)